src/main/webapp/WEB-INF/views/index.jsp

```diff
@@ -16,16 +16,20 @@
 
     <div class="container">
       <div class="columns">
-
+        <img src="lowcarbdiet.jpg" width="600" height="310">
```

Comment:
i think the image height and width stuff should be scaled instead of hardcoded, so that on things like mobile the picture won't be too big